stock_inventory_line_price/views/inventory_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <record id="view_inventory_form" model="ir.ui.view">
+        <field name="name">view.inventory.form.inherit</field>
+        <field name="model">stock.inventory</field>
+        <field name="inherit_id" ref="stock.view_inventory_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='line_ids']//field[@name='product_id']" position="after">
+                <field name="theoretical_std_price"/>
+                <field name="standard_price"/>
+            </xpath>
+            <xpath expr="//tree" position="attributes">
+                <attribute name="colors">blue: product_qty != theoretical_qty or theoretical_std_price != standard_price; red: theoretical_qty &lt; 0</attribute>
```

Comment:
Wow I never considered the modularity concerns of changing line colors - great catch.

What if we conformed to some sort of interface mechanism with a field and a method? Something like:

```
class StockInventory(Model):
    _inherit = 'stock.inventory'

    attribute_colors = fields.Text(compute='_compute_attribute_colors')

    def _compute_attribute_colors(self):
        colors = '; '.join(["%s: %s" % (k, v) for k, v in colors.items()])
        for record in self:
            record.attribute_colors = colors

    @api.model
    def _get_attribute_colors(self):
        return {
            'blue': 'product_qty != theoretical_qty or theoretical_std_price != standard_price',
            'red': 'theoretical_qty &lt; 0',
        }
```

We'd need a base module though.